packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -70,7 +70,7 @@ export default {
 
     data () {
         return {
-            panelCollapsed: false,
+            panelCollapsed: true,
```

Comment:
Out of interest – what happens if JS never loads? Do the panels stay collapsed, or are they open in that scenario?

Reply:
Wouldn't that completely fail to load the Vue content? Not sure how that works for SSR...

Reply:
Yeh, it's SSR that I'm thinking about. I think that's why we have the default state set to false atm, because we'd want those links to be visible if the client JS failed to load for some reason (and some search bots may not parse the JS correctly).

Reply:
One way to get around that would be to revert back to `false` and set on mount based on screen size. However that would need to re-introduce screen resize listener for responsive... Let me have a look again...